packages/feature-detector/package.json

```diff
@@ -13,14 +13,29 @@
   "bugs": {
     "url": "https://github.com/vercel/edge-runtime/issues"
   },
-  "keywords": [],
+  "keywords": [
+    "detector",
+    "edge-runtime",
+    "edge",
+    "feature",
+    "functions",
+    "runtime",
+    "standard",
+    "web"
+  ],
+  "dependencies": {
+    "ts-morph": "17.0.1"
+  },
   "devDependencies": {
     "jest-text-transformer": "1.0.4",
-    "next": "^13.1.1",
-    "react": "~18.2.0",
-    "react-dom": "~18.2.0",
-    "tsup": "^6"
+    "next": "13",
+    "react": "18",
+    "react-dom": "18",
+    "tsup": "6"
```

Comment:
**question:** why freezing the **dev** dependencies? I meant to be working on latest version. While I understand freezing dependencies brings more reproducible builds, I'm not sure to see the benefit for dev deps.

On the other hand, it's usefull to test against latest next.js and early catch bugs.

Reply:
Since there aren't super important, all of them are locked into the major version just to avoid dependabot PR noise.

If a new major version of Next.js is released, then we will have the PR upgrading it 🙂